lib/controller.js

```diff
@@ -946,11 +946,22 @@ controller.tesselFirmwareVerion = function(opts) {
         return updates.requestBuildList().then(function(builds) {
           // Figure out what commit SHA is running on it
           var version = updates.findBuild(builds, 'sha', versionSha).version;
-          logs.info('Tessel [' + tessel.name + '] version: ' + version);
+          var tesselVersion = 'Tessel [' + tessel.name + '] version: ' + version;
+
+          return tessel.fetchCurrentNodeVersion()
+            .then(nodeVersion => {
+              logs.info(tesselVersion);
+              logs.info('Tessel [' + tessel.name + '] Node version: ' + nodeVersion);
+            })
+            .catch(() => {
+              logs.info(tesselVersion);
+              logs.info('Tessel [' + tessel.name + '] Node version: unknown');
+            });
         });
       })
-      .catch(function() {
+      .catch(() => {
         logs.info('Tessel [' + tessel.name + '] version: unknown');
+        logs.info('Tessel [' + tessel.name + '] Node version: unknown');
       });
   });
 };
```

Comment:
This is part of a bigger issue that can be addressed once I've finished the migration to `npmlog`. When that's done I will call on you to help with this. Basically the idea is that other programs should be able to invoke these commands from spawn or exec and get useful and easily consumable information from them—but like I said, that needs to be on hold until after I finish the move to npmlog